test/integration/customUrlRest.spec.ts

```diff
@@ -0,0 +1,16 @@
+import CustomUrlRestClient, { CustomUrlRestClientRestClient } from "./generated/customUrlRest/src";
+import { assert } from 'chai';
+
+describe("Custom Endpoint", () => {
+  let client: CustomUrlRestClientRestClient;
+  let clientOptions: any;
+  beforeEach(() => {
+    clientOptions = { host: "host:3000", allowInsecureConnection: true };
```

Comment:
From what I see in the client options, there is no `host` property. We have only `baseUrl`. How are you setting this property? I think this  might be the reason the CI is failing with the error message:

```
TypeError [ERR_INVALID_URL]: Invalid URL: http://{accountName}[object Object]//customuri
```

Could you please check and fix it? Does this code run locally?

Reply:
yeah, I am still trying to understand why